### PR TITLE
dut_pwr: display notifcations on power failure

### PR DESCRIPTION
**Pull request by hnez**
Add notification on the LCD …

lcd

… and web interface if the LXA TAC's DUT Power supply shut off due to e.g. an overcurrent or overvoltage event.

web

### CODE REVIEW

File: web/src/TacComponents.tsx
Comment (Emantor) [inline web/src/TacComponents.tsx:520]: Nit: maybe we should add a default handler in case none of the cases match. Otherwise looks good to me.

**Review by Emantor — APPROVED** [review]

File: web/src/TacComponents.tsx
Comment (hnez) [reply web/src/TacComponents.tsx:553]: I initially agreed but I've had a second look and think we shouldn't. In the default case we would want `reason` to be `null` to make sure the `Alert` is set to be not visible:

https://github.com/linux-automation/tacd/blob/eb97382b14cbbab2e6cba0dd9ca4ac118187368d/web/src/TacComponents.tsx#L555-L562

This is however already the case when none of the `case`s match, as `reason` is initialized that way.

**Comment by hnez** [comment]
I've resolved merge conflicts in `src/ui/screens.rs` and `web/src/TacComponents.tsx` caused by other alert screens added to the `main` branch in the meantime.
Please re-approve and merge if you like.

**Comment by hnez** [comment]
Hi @KarlK90 can I get a new approval? The previous one was invalidated by required changes in a rebase.

**Review by KarlK90 — APPROVED** [review]

**Pull request merged**
